src/App.xaml.cs

```diff
@@ -149,6 +150,8 @@ public App()
             services.AddTransient<ShellViewModel>();
             services.AddTransient<WhatsNewViewModel>();
             services.AddTransient<InitializationViewModel>();
+            services.AddTransient<RepositoryManagementMainPageViewModel>();

```

Comment:
Consider making an extension method for the repo management so from the `App.xaml.cs` we can just call.
```
services.AddRepoManagement()
```

This will maintain an abstract level of information about repo management from the App perspective, reduces potential code conflicts and reduces the domain scope of file changes in future PRs (Repo management changes will "only" be in the Repo management csproj).

Reply:
Yes.  Correct.  I will when I add more viewmodels.  However, even if I add no more view models, I will move this to it's own method.

Reply:
Fixed.